src/services/twilio-api/get-flag-config.js

```diff
@@ -0,0 +1,31 @@
+const { kebabCase } = require('@twilio/cli-core').services.namingConventions;
+const { BASE_TOPIC_NAME } = require('./get-topic-name');
+
+// AccountSid is a special snowflake
+const ACCOUNT_SID_FLAG = 'AccountSid';
+
+const getFlagName = paramName => {
+  return kebabCase(
+    paramName
+      .replace('<', 'Before')
+      .replace('>', 'After')
+  );
+};
+
+const getFlagConfig = (parameter, actionDefinition) => {
```

Comment:
Suggest adding a one-line class description similar to the PR commit summary:
a config utility for converting API params to CLI flags